osquery/config/packs.cpp

```diff
@@ -74,9 +74,9 @@ size_t getMachineShard(const std::string& hostname = "", bool force = false) {
   auto hn_hash = getBufferSHA1(hn.c_str(), hn.size());
 
   if (hn_hash.size() >= 2) {
-    long hn_char;
-    if (safeStrtol(hn_hash.substr(0, 2), 16, hn_char)) {
-      shard = (hn_char * 100) / 255;
+    auto const hn_char = tryTo<long>(hn_hash.substr(0, 2), 16);
```

Comment:
This variable contains the type name `char` but it is `tryTo<long>`

Reply:
Fixed name `hn_char` to `hn_num`, thanks!